pycon/management/commands/update_tutorial_registrants.py

```diff
@@ -79,21 +80,21 @@ def handle_noargs(self, **options):
                 tutorial.save()
 
                 try:
-                    user = User.objects.get(email=user_email)
+                    user = User.objects.get(id=int(user_id))
```

Comment:
It looks like `int(user_id)` should require a new `except` clause, in case `user_id` is not formatted as an integer.